docs/spire.md

```diff
@@ -64,7 +64,7 @@ When a TaskRun is created:
 ## Enabling TaskRun result attestations
 
 To enable TaskRun attestations:
-1. Make sure `enforce-nonfalsifiability` is set to `"spire"` and `enable-api-fields` is set to `"alpha"` in the `feature-flags` configmap, see [`install.md`](./install.md#customizing-the-pipelines-controller-behavior) for details
+1. Make sure `enforce-nonfalsifiability` is set to `"spire"`. See [`additional-configs.md`](./additional-configs.md#customizing-the-pipelines-controller-behavior) for details
```

Comment:
NIT: It looks like that the `enforce-nonfalsifiablity` is neither set in `additonal-configs/customizing-the-pipelines-controller-behavior` section, is it WIP? 🤔

Reply:
yes, spire is still non-functional